CHANGELOG.md

```diff
@@ -20,7 +20,7 @@ Given a version number `MAJOR.MINOR.PATCH`, we increment the:
 - Backward compatibility in versions `0.0.z` is **not guaranteed** when `z` is increased.
 - Backward compatibility in versions `0.y.z` is **not guaranteed** when `y` is increased.
 
-## Unreleased
+## v0.10.5
 
 ### Added
```

Comment:
Nitpick, but you name it "sharing outputs" feature here, whereas in previous changelog entries, it's `outputs-sharing` experiment. Would be better to have some consistency in the naming.